404.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: 404
+permalink: /404.html
+---
+
+<script type="text/javascript">
+    // Redirect to the homepage after 5 seconds
+    setTimeout(function() {
+      window.location.href = 'https://osaa.dk';
+    }, 5000);
```

Comment:
An automatic redirect is not necessary. It's fine with a 404 page and some nice text, but the page should just include a link to the front page.
Automatic redirect does no good for us, SEO wise. If the response code was already 404, Google will exclude the URL from their index anyway. This would only add a bit of confusion for the user, and possibly do some actual work on poorly written crawler bots. But then again, it might do some harm, on other poorly written crawler bots.

I'd suggest for us to keep the 404 page, but remove the automatic redirect.